tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -200,6 +200,8 @@ var _ = ginkgo.Describe("VM backup and restore tests", ginkgo.Ordered, func() {
 		gomega.Expect(err).To(gomega.BeNil())
 		err = v.CreateWaitForFirstConsumerStorageClass("test-sc-wffc")
 		gomega.Expect(err).To(gomega.BeNil())
+		lib.DeleteBackupRepositoryByRegex(runTimeClientForSuiteRun, namespace)
+		gomega.Expect(err).To(gomega.BeNil())
```

Comment:
this is wrong, it checking error from line 201

but check my previous comment, it also fixes this

Reply:
fixed, thank you!

Reply:
thanks updated.